game_life.js

```diff
@@ -7,6 +7,20 @@ window.onload = function() {
 	document.getElementById("mySpan").innerHTML = height + "x" + width;
 }
 
+function myDisabled() {
+	document.getElementById("button1").disabled = true;
+    document.getElementById("button2").disabled = false;
+	document.getElementById("button1").style.cursor = "not-allowed";
+	document.getElementById("button1").style.opacity = 0.6;
+	start();
+}
+function myDisabled2() {
+	document.getElementById("button2").disabled = true;
```

Comment:
1,3,4 rows of myDisabled are the same as 1,2,3 rows of myDisabled2, only name varies. So, create separate function for these three lines. And call it with name as parameter